hatt_classifier.py

```diff
@@ -179,14 +179,14 @@ def prepare_embeddings(wrd2id):
 			dtype='float32')
 		embedding_idx[wrd] = coefs
 	glove_f.close()
-	print "Found %s word vectors." % len(embedding_idx)
+	print("Found %s word vectors." % len(embedding_idx))
 
 
 	embedding_mat = np.random.rand(vocab_size+1,EMBEDDING_DIM)
 
 	wrds_with_embeddings = 0
 	# Keep the MAX_NB_WORDS most frequent tokens.
-	for wrd, i in wrd2id.items():
+	for wrd, i in list(wrd2id.items()):
```

Comment:
this is already a list

Reply:
As mentioned in the other comment, this type changed between Python 2 and 3. But since this is using the iteration protocol changing it to your suggestion works and is what I've done in 
fc2a627